x-pack/test/accessibility/apps/group1/management.ts

```diff
@@ -52,27 +53,23 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
             it('index details - overview', async () => {
               await PageObjects.settings.clickIndexManagement();
               await PageObjects.indexManagement.clickIndexAt(0);
+              await PageObjects.searchIndexDetail.expectIndexDetailsPageIsLoaded();
               await a11y.testAppSnapshot();
             });
 
             it('index details - settings', async () => {
-              await PageObjects.indexManagement.clickIndexDetailsTab('settings');
+              await PageObjects.searchIndexDetail.changeTab('settingsTab');
```

Comment:
Can we update the tab key to be just `settings` and not `settingsTab` ? (not required for this PR, but something we should note)

Reply:
In this case, we need to change all other tabs as well for consistency, and I'm afraid it requires additional work for this pr and may slow down a bit merging